image_writers_guide/guidelines.adoc

```diff
@@ -5,143 +5,186 @@
+*Avoiding SSHD*
 
-=== Do not start (or install) SSHD
-Running SSHD should be avoided.  For accessing running containers you can use `docker exec` locally, or the OpenShift tooling which allows execution of arbitrary commands in running images.  Installing and running SSHD in your image opens up additional vectors for attack and requirements for security patching.
+Do not start or install SSHD in your image. For accessing running containers, you can use the `docker exec` command locally, or the OpenShift tooling which allows execution of arbitrary commands in running images. Installing and running SSHD in your image opens up additional vectors for attack and requirements for security patching.
 
-=== Use Volumes for Persisted Data
-Images should expect to use a https://docs.docker.com/reference/builder/#volume[Docker volume] for persisted data.  This way the OpenShift system can mount network storage to the node where the container is running and then reattach that storage at a new node if the container moves.  By using the volume for all persisted storage needs, the content will be preserved throughout container restarts and moves.  If your image writes data to arbitrary locations within the container, there is no guarantee that content will be preserved.
+*Using Volumes for Persisted Data*
 
-All data which is expected to live longer than the container must be written to a volume.  With Docker 1.5 there will be a `readonly` flag for containers which can be used to strictly enforce good practices about not writing data to ephemeral storage in a container, so designing your image around that capability now will make it easier to take advantage of it later.
+Images should expect to use a https://docs.docker.com/reference/builder/#volume[Docker volume] for persisted data. Doing so allows OpenShift to mount network storage to the node where the container is running, then reattach that storage at a new node if the container moves. By using the volume for all persisted storage needs, the content is preserved throughout container restarts and moves. If your image writes data to arbitrary locations within the container, there is no guarantee that content will be preserved.
 
-Furthermore, by explicitly defining volumes in your Dockerfile, you make it easy for consumers of the image to understand what volumes they need to define when running your image.
+All data which is expected to live longer than the container must be written to a volume. With Docker 1.5, there will be a `readonly` flag for containers which can be used to strictly enforce good practices about not writing data to ephemeral storage in a container. Designing your image around that capability now will make it easier to take advantage of it later.
```

Comment:
[Suggestion]
All data which is expected to live longer than the container must be written to a volume. ==>> All data that you want to preserve even after the container is destroyed must be written to a volume.